mesh/mesh.sh

```diff
@@ -28,44 +26,39 @@ create_token() {
 
 authorization() {
   curl "${CURL_FLAGS}" -X POST "https://${HOST}/messageexchange/${MAILBOX_ID}" \
-    --cert "${OPENTEST_ENDPOINT_CERT}" --key "${OPENTEST_ENDPOINT_PRIVATE_KEY}" -H "Authorization: ${TOKEN}" \
+    --cert "${MESH_ENDPOINT_CERT}" --key "${MESH_ENDPOINT_PRIVATE_KEY}" -H "Authorization: ${TOKEN}" \
     -H 'Mex-ClientVersion: 1.0' -H 'Mex-JavaVersion: 1.7.0_60' -H 'Mex-OSArchitecture: Windows 7' -H 'Mex-OSName: x86' -H 'Mex-OSVersion: 6.1'
 }
 
 inbox() {
   curl "${CURL_FLAGS}" -X GET "https://${HOST}/messageexchange/${MAILBOX_ID}/inbox" \
-    --cert "${OPENTEST_ENDPOINT_CERT}" --key "${OPENTEST_ENDPOINT_PRIVATE_KEY}" -H "Authorization: ${TOKEN}" \
+    --cert "${MESH_ENDPOINT_CERT}" --key "${MESH_ENDPOINT_PRIVATE_KEY}" -H "Authorization: ${TOKEN}" \
     -H 'Mex-ClientVersion: 1.0' -H 'Mex-JavaVersion: 1.7.0_60' -H 'Mex-OSArchitecture: Windows 7' -H 'Mex-OSName: x86' -H 'Mex-OSVersion: 6.1'
 }
 
 send() {
   local body
-  local count
   body="$1"
-  count="${2:-1}"
-  for ((i = 1 ; i <= $count ; i++)); do
```

Comment:
This is required for the performance testing, how come you're reverting that?

Reply:
Fro some reason a loop inside the script didnt work. 1st request pass and the rest fails. Dunno why. So I've moved if outside and described in the vnp readme

Reply:
Probably because the token is single-use. You'd need to create a new token for each curl